tests/Jenkins/utilities.sh

```diff
@@ -332,6 +332,26 @@ function diracInstallCommand(){
 	./dirac-install -r `cat dirac.version` -t server -d
 }
 
+#.............................................................................
+#
+# prepareForServer:
+#
+#   This function gets the DIRAC install script
+#
+#.............................................................................
+
+function prepareForServer(){
+	echo '==> [prepareForServer]'
+
+	#Move to server directory
+	cd $SERVERINSTALLDIR
```

Comment:
Mmm I am not sure what you mean. : $SERVERINSTALLDIR contains a full 
path (also $PILOTINSTALLDIR and $TESTCODE)